sdk/ml/azure-ai-ml/azure/ai/ml/_utils/utils.py

```diff
@@ -641,14 +807,36 @@ def convert_identity_dict(
 
 
 def strip_double_curly(io_binding_val: str) -> str:
+    """Strip double curly brackets from a string.
```

Comment:
Do CI force you to add docstring for all functions in changed file? LoL

Reply:
hhh actually it's the interrogate in pre-commit. Thank copilot for helping on this